programmers/난이도별/level01.문자열_다루기_기본/sjjyy.java

```diff
@@ -0,0 +1,26 @@
+package 난이도별.level01.문자열_다루기_기본;
+
+public class sjjyy {
+    public static boolean solution(String s)
+    {
+        int len = s.length();
+
+        if (len == 4 || len == 6)
+        {
+            try {
+                Integer.parseInt(s); // 문자열 -> 정수
+                return true;
+            } catch (NumberFormatException e) { // 숫자 형식 오류
+                return false;
+            }
```

Comment:
```suggestion
            return s.matches("[0-9]");
```
정규식을 이용하도 된답니다!

https://codechacha.com/ko/java-string-matches/